### PR TITLE
GetFixedAsString for decimal

### PR DESCRIPTION
**Pull request by NovichikhinAlexey**
GetFixedAsString for decimal and use fixed accuracy format with InvariantCulture

### CODE REVIEW

File: src/Lykke.Common/Utils.cs [hunk 1]
```diff
@@ -929,7 +929,12 @@ public static double Map(this double value, double minFrom, double maxFrom, doub
 
         public static string GetFixedAsString(this double value, int accuracy)
         {
-            return value.ToString("0." + new string('#', accuracy));
+            return value.ToString("0." + new string('0', accuracy), CultureInfo.InvariantCulture);
```
Comment (MikhailGerasimov) [inline src/Lykke.Common/Utils.cs]: What if double = NaN?

File: tests/Common.Tests/UtilsClass/FloatOperations.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+﻿using Xunit;
+
+namespace Common.Tests.UtilsClass
+{
+    public class FloatOperations
+    {
+        [Fact]
+        public void GetFixedAsString()
+        {
```
Comment (MikhailGerasimov) [inline tests/Common.Tests/UtilsClass/FloatOperations.cs]: No test for double

File: src/Lykke.Common/Utils.cs (hunk 1 above)
Comment (KonstantinRyazantsev) [reply src/Lykke.Common/Utils.cs]: Use string interpolation instead of "+" operator

File: src/Lykke.Common/Utils.cs [hunk 3]
```diff
@@ -929,7 +929,12 @@ public static double Map(this double value, double minFrom, double maxFrom, doub
...
+        }
+
+        public static string GetFixedAsString(this decimal value, int accuracy)
+        {
+            return value.ToString("0." + new string('0', accuracy), CultureInfo.InvariantCulture);
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Common/Utils.cs]: Use string interpolation instead of "+" operator

**Pull request merged**